my-cart/my-cart.css

```diff
@@ -0,0 +1,154 @@
+@import "/common.css";
+
+.body-container{    
+    display: flex;
+    align-items: center;
+    margin-top: 7rem;
+    justify-content: center;
+}
+
+
+.my-cart-2 {
+    text-align: center;
+    margin: 2rem 0;
+}
+
+/*          Page Layout */
+
+.product-detail {
+    display: flex;
+    gap: 10px;
+    width: 50rem;
+}
+
+.nikon-img {
+    width: 40%;
+}
+
+.my-cart-detail{
+    background-color: skyblue;
+    display: flex;
+    flex-direction: column;
```

Comment:
Suggestion: You can consider using CSS variables for properties like `background-color` and `font-color`.

Rest all looks good!

Reply:
ok, I will do it.